.github/workflows/unit-tests-code.yaml

```diff
@@ -38,6 +38,10 @@ jobs:
         TELEPORT_XAUTH_TEST: yes
         TELEPORT_BPF_TEST: yes
         WEBASSETS_SKIP_BUILD: 1
+
+        # ./integrations/lib/testing/integration
+        TELEPORT_GET_VERSION: v12.1.0
+        TELEPORT_ENTERPRISE_LICENSE: ${{ secrets.TELEPORT_ENTERPRISE_LICENSE }}
```

Comment:
So the problem with this is:
* The access plugin "integration tests" themselves live in `integrations/access/slack`. So they are not treated as integration tests by Makefile/GHA
* The plugin integration-testing *supporting code* lives in `integrations/lib/testing/integration`. This supporting code has some self-tests. Due to that last folder being named `integration`, these self-tests run under "integration tests" in GHA.

This is not quite as intended, but I'd like to avoid more reshuffling of these packages for now - would like this initial PR to be a straightforward copy-paste mostly.